src/netif.c

```diff
@@ -1952,8 +1993,8 @@ static inline lcoreid_t get_master_xmit_lcore(void)
     static int i = 0;
     lcoreid_t cid;
 
-    if (lcore_conf[i].nports > 0) {
-        cid = lcore_conf[i].id;
+    cid = lcore_conf[i].id;
+    if (netif_lcore_is_fwd_worker(cid)) {
         i++;
     } else {
         cid = lcore_conf[0].id;
```

Comment:
i = 0;
if (netif_lcore_is_fwd_worker(lcore_conf[0].id) == false ) {
    cid = lcore_conf[0].id;
}

lcore_conf[0] mast be type LCORE_ROLE_FWD_WORKER